elyra/pipeline/processor_kfp.py

```diff
@@ -163,10 +163,14 @@ def export(self, pipeline, pipeline_export_format, pipeline_export_path, overwri
                                operation.inputs,
                                operation.outputs)
 
+            # TODO
+            wcd = os.getenv('ELYRA_WRITABLE_CONTAINER_DIR', '/tmp').rstrip('/')
```

Comment:
Should we make the `/tmp` default value be the result of a method like `TempDirectory()` or whatever the platform-independent function is?

Reply:
In this case we unfortunately cannot, because the location is specific to the runtime container image where the bootstrapper will be executed. The issue is similar to the one we've discussed in the related KFP-notebook PR.

Reply:
Right - I recall that now - thanks.  That makes sense and `/tmp` is the most reasonable default in this case.